src/modules/News/controller.ts

```diff
@@ -78,4 +78,32 @@ const getTopHeadlines = async (req: Request, res: Response) => {
   }
 };
 
-export default { getLatestNews, getTopHeadlines, getPersonalizedNews };
+const fetchSingleArticle = async (req: Request, res: Response) => {
+  const { title, description, publishedAt } = req.query;
+
+  try {
+    const article = await fetchFromNewsAPI('/everything', {
```

Comment:
I did not get it. Are you still calling the fetchFromNewsAPI. Should not you call the fetchSingleArticle from utils? And also import it here in this file.

Reply:
the method to fetch the single article reuses the method fetchijng everything but adds the params.